### PR TITLE
DEV --> PROD

### PR DESCRIPTION
**Pull request by nkewden**
Please take at look at our mergre from development to production

### CODE REVIEW

File: capstone-ui/src/components/BrowseDetail/BrowseDetail.jsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default function BrowseDetail(props) {
         const {data, err} = await apiClient.fetchDonationById(donationId)
         if(err) setError(err)
         if(data){
+          console.log('data: ', data);
```
Comment (Rebecca-CYLi) [inline capstone-ui/src/components/BrowseDetail/BrowseDetail.jsx]: found a console log

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
found a console log, please fix it

File: capstone-ui/src/components/BrowseDetail/BrowseDetail.jsx (hunk 1 above)
Comment (LifeBringer) [reply capstone-ui/src/components/BrowseDetail/BrowseDetail.jsx]: What she said

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See Rebecca's comment.

File: capstone-ui/src/components/BrowseDetail/BrowseDetail.jsx (hunk 1 above)
Comment (nkewden) [reply capstone-ui/src/components/BrowseDetail/BrowseDetail.jsx]: just removed it!

**Review by Rebecca-CYLi — APPROVED** [review]
console log removed, LGTM

**Pull request merged**